app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,25 @@
+@import "base/fonts";
```

Comment:
try to remove it and see what it does!

SPOILER: If its loaded no need to `@import` it again